testbed/lib/main.dart

```diff
@@ -49,8 +49,9 @@ void main() {
       final top = ((screenFrame.height - height) / 3).roundToDouble();
       final frame = Rect.fromLTWH(left, top, width, height);
       window_size.setWindowFrame(frame);
-      window_size
-          .setWindowTitle('Flutter Testbed on ${Platform.operatingSystem}');
+      final title = 'Flutter Testbed on ${Platform.operatingSystem}, こんにちは世界';
```

Comment:
Please remove the test string.

(If `SetWindowText` were calling the `A` variant rather than the `W` variant the code wouldn't have compiled with a std::wstring, FYI.)

Reply:
I'll happily remove it once I can get it to display on Windows. Right now it's broken.

Reply:
Removed